hadoop-ozone/dist/src/main/compose/common/grafana/dashboards/Ozone - ListKey Metrics.json

```diff
@@ -77,7 +69,7 @@
     {
       "datasource": {
         "type": "prometheus",
-        "uid": "${DS_PROMETHEUS-1}"
+        "uid": "${DS_PROMETHEUS}"
```

Comment:
`Datasource ${DS_PROMETHEUS} was not found`

I have checked out the patch locally and tried viewing the dashboards using Graphana via a docker-compose cluster with the monitoring add-ons, the variable: `DS_PROMETHEUS` does not get resolved automatically - not sure if it has something to do with my local settings! (please find the image attached).

![Dashboards Error](https://github.com/apache/ozone/assets/46785609/b7259f15-8b76-4795-9b77-a012e18f089e)

I tried removing the `uid` field under `datasource` after which I do not run into this issue - I get to see the graphs as expected.
```
"datasource": {
  "type": "prometheus"
},
```

Reply:
Did you import it as a new dashboard?

Reply:
While adding a new dashboard (by importing the `.json` file) and selecting the Prometheus source, I can see the graphs. However, when attempting to view graphs for the existing dashboards, I encounter the error message: `Datasource ${DS_PROMETHEUS} was not found`.

Reply:
I was able to reproduce the issue. It only impacts the pre provisioned Dashboards in the dev environment but should be ok for any customer importing a dashboard. I will update with the fix that makes it work for both.

Reply:
Done

Reply:
Dear ozone comiters, 
The latest Grafana is depreciating certain visual currently used within the current object-metrics dashboard.
I've updated upon recent expectation hope this helps.

[Ozone  - Object Metrics update dahsboards.json](https://github.com/apache/ozone/files/14723282/Ozone.-.Object.Metrics.update.dahsboards.json)

Reply:
Thanks @PhilippeLan will include these changes.